test/src/receiver/main.cpp

```diff
@@ -18,7 +30,6 @@
 #include "../../../lib/vendor/iestream.hpp"

```

Comment:
Hi,

Please make all the file paths in all source files softcoded. (main of ABP, Sender, Subnet, Receiver) 

Thanks

Reply:
what do you mean by softcoded

Reply:
Hi,
The paths can be softcoded with the help of "define", so when the path of the inputs changes, it will be helpfull for user with minimal changes i.e only in "define" rather than changing it all the places.
 please refer the below images
![image](https://user-images.githubusercontent.com/46407088/63179703-bcc1a200-c01a-11e9-990e-67cafa0af6fe.png)

![image](https://user-images.githubusercontent.com/46407088/63179872-14f8a400-c01b-11e9-947b-216774e9cf3f.png)

Reply:
done